app/services/guidance_link_service.rb

```diff
@@ -0,0 +1,85 @@
+class GuidanceLinkService
+  class << self
+    def call(document:, chapter: nil, just_the_page_number: false)
+      feature_flag = FeatureFlags.enabled?(:mtr_phase_1, without_session_data: true) ? :mtr_phase_1 : :pre_mtr_phase_1
+
+      if just_the_page_number
+        mapping[feature_flag].dig(document, :sections, chapter)
+      elsif chapter
+        mapping[feature_flag].dig(document, :page_url).to_s << "#page=" << mapping[feature_flag].dig(document, :sections, chapter).to_s
+      else
+        mapping[feature_flag].dig(document, :page_url)
+      end
+    end
+
+    def mapping
+      {
+        pre_mtr_phase_1: {
+          mental_heatlh_guidance: {
+            page_url: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1143984/Mental_Health_Guidance_-_Contract_management_-_GOV.UK_2023.pdf",
+          },
+          lc_guidance_controlled: {
+            page_url: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1157029/Lord_Chancellor_s_guide_to_determining_financial_eligibility_for_controlled_work_and_family_mediation_May_2023.pdf",
+            sections: {
+              legacy_guidance: "3",
```

Comment:
The page numbers are numbers, so we might as well represent them as numbers, i.e. `legacy_guidance: 3`